test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -3188,4 +3188,22 @@ function Test-InstallPackageWithEscapedSymbolInPath()
 
     # Assert
     Assert-Package $p Xam.Plugin.Connectivity
+}
+
+function Test-InstallPackageWithRootNamespaceInPPFile {
```

Comment:
how about we start adding new e2e tests via apex instead?

Reply:
I think it's a good idea, but we need to discuss it as a team before going that route, because then it will require more time to add tests (at least initially) and we may need to include that as well in our costing.